subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_shi_ite.c

```diff
@@ -66,17 +64,21 @@ struct shi_it8xxx2_cfg {
 
 struct shi_it8xxx2_data {
 	/* Peripheral data */
-	struct k_sem handler_owns;
-	struct k_sem dev_owns;
+	struct ec_host_cmd_rx_ctx *rx_ctx;
+	struct ec_host_cmd_tx_buf *tx;
 	struct gpio_callback cs_cb;
 	/* Current state */
 	enum shi_state_machine shi_state;
 	/* Buffers */
-	uint32_t in_msg_size;
 	uint8_t in_msg[SPI_RX_MAX_FIFO_SIZE] __aligned(4);
 	uint8_t out_msg[SPI_TX_MAX_FIFO_SIZE] __aligned(4);
 };
 
+struct ec_host_cmd_shi_ite {
```

Comment:
It should have some suffix to inform if this is config, context or something else

Reply:
Good point. Added.